sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -144,18 +125,15 @@ public virtual async Task<RecognizeContentOperation> StartRecognizeContentFromUr
         /// </summary>
         /// <param name="receiptFileStream">The stream containing the one or more receipts to recognize values from.</param>
         /// <param name="receiptLocale"></param>>
-        /// <param name="contentType"></param>
         /// <param name="recognizeOptions"></param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>A <see cref="RecognizeReceiptsOperation"/> to wait on this long-running operation.  Its <see cref="RecognizeReceiptsOperation"/>.Value upon successful
         /// completion will contain the extracted receipt.</returns>
         [ForwardsClientCalls]
-        public virtual async Task<RecognizeReceiptsOperation> StartRecognizeReceiptsAsync(Stream receiptFileStream, ContentType contentType, string receiptLocale = "en-US", RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
+        public virtual async Task<RecognizeReceiptsOperation> StartRecognizeReceiptsAsync(Stream receiptFileStream, string receiptLocale = "en-US", RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
```

Comment:
is this public API? I thought we removed the `receiptLocale` for this preview.

Reply:
You're right. It was left by mistake. I'll remove it.